Assets/Plugins/iOS/mParticleUnity.m

```diff
@@ -530,16 +530,16 @@ + (MPIdentityApiRequest *) MPIdentityApiRequest:(NSDictionary *)identityRequestD
     if ([[identityRequestDict allKeys] containsObject:@"UserIdentities"]) {
         NSDictionary *identities = identityRequestDict[@"UserIdentities"];
         for (id key in identities) {
-            [identityRequest setUserIdentity:identities[key] identityType:(MPUserIdentity)[key integerValue]];
+            [identityRequest setIdentity:identities[key] identityType:(MPUserIdentity)[key integerValue]];
```

Comment:
No visible @interface for 'MPIdentityApiRequest' declares the selector 'setUserIdentity:identityType:'